aiida/cmdline/commands/cmd_config.py

```diff
@@ -182,6 +182,8 @@ def verdi_config_caching(disabled):
     from aiida.manage.caching import get_use_cache
     from aiida.plugins.entry_point import ENTRY_POINT_STRING_SEPARATOR, get_entry_point_names
 
+    echo.echo_deprecated('This command is deprecated, please use set/unset to config the central `config.json` file.')
```

Comment:
I would use the dedicated decorator we have `aiida.cmdline.utils.decorators.deprecated_command`. And maybe for the text we could make the command more explicit and I would remove reference to the `config.json`. Also, this command was only used for listing, not for setting caching options, so I would remove that. So something like:
```
This command has been deprecated and will be removed. Please use `verdi config list caching` instead.
```